### ISSUE TITLE
strange error

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#3420]
I don't know what's causing this, but when moving with my cursor, I get a lot of these errors.

```
  | cropImage | @ | content-crop.js:115
-- | -- | -- | --
  | showCoords | @ | content-crop.js:240
  | update | @ | jquery.Jcrop.js:1002
  | selectDrag | @ | jquery.Jcrop.js:275
  | trackMove | @ | jquery.Jcrop.js:1178
  | dispatch | @ | jquery.js:5183
  | elemData.handle | @ | jquery.js:4991
```

![screenshot from 2018-12-14 11-21-22](https://user-images.githubusercontent.com/207405/49997697-92eb4380-ff92-11e8-993e-4862786ec430.png)

**Comment by williamstein** [comment]
That looks like that horrible library for cropping avatar images that was added this summer...  Last time I looked at it I, I wanted to remove it from cocalc for code quality issues -- even writing something from scratch would probably be better...

**Comment by haraldschilly** [comment]
Yes, also my first thought. I don't know what I did to trigger it... I haven't seen that error again yet either. I still found it worth to report.
